ONSdigital/census-rm-case-api-2021
# Copied updated data model from case-processor and return new events

# Motivation and Context

We want to be able to see the print file selected events for a case.

# What has changed



Data model copied from case-processor and updated DTO builder to be able to return events associated directly with a case.

# How to test?



Run with case-processor on branch update-case-history and the action scheduler on branch case-history-updates
Retrieve case details using option ?caseEvents=true and check "PRINT_CASE_SELECTED" is in the list of events.

# Links



https://trello.com/c/stICnQSn/922-rmc-212-case-processor-update-case-history-when-added-to-print-file-13
# Screenshots (if appropriate):
